app/javascript/pages/forgot-password.tsx

```diff
@@ -33,14 +33,15 @@ const ForgotPassword = () => {
         <div className="form-card__lead text-center border-b mx-0">
           <Icon size="2xl" symbol="profile" />
           <h2 className="form-card__title">{t("pageTitle.forgotPasswordLowercase")}</h2>
+          <p className="form-subtitle">{t("forgotPassword.subtitle")}</p>
         </div>
         {!emailSubmitted ? (
           <div className="form-card__group pt-0 border-b">
```

Comment:
issue: probably overlooked in a previous pr- the `border-b` style creates a non-smooth edge on the bottom of the card
![Screenshot 2024-12-31 at 1 09 55 PM](https://github.com/user-attachments/assets/335134ae-8017-44e3-9861-61818f327343)